tests/mir-opt/lower_intrinsics.read_via_copy_primitive.LowerIntrinsics.diff

```diff
@@ -4,17 +4,17 @@
   fn read_via_copy_primitive(_1: &i32) -> i32 {
       debug r => _1;                       // in scope 0 at $DIR/lower_intrinsics.rs:+0:32: +0:33
       let mut _0: i32;                     // return place in scope 0 at $DIR/lower_intrinsics.rs:+0:44: +0:47
-      let mut _2: *const i32;              // in scope 0 at $DIR/lower_intrinsics.rs:+1:46: +1:47
+      let mut _2: &i32;                    // in scope 0 at $DIR/lower_intrinsics.rs:+1:46: +1:47
       scope 1 {
       }
   
       bb0: {
           StorageLive(_2);                 // scope 1 at $DIR/lower_intrinsics.rs:+1:46: +1:47
-          _2 = &raw const (*_1);           // scope 1 at $DIR/lower_intrinsics.rs:+1:46: +1:47
--         _0 = read_via_copy::<i32>(move _2) -> [return: bb1, unwind unreachable]; // scope 1 at $DIR/lower_intrinsics.rs:+1:14: +1:48
+          _2 = _1;                         // scope 1 at $DIR/lower_intrinsics.rs:+1:46: +1:47
+-         _0 = read_via_copy::<&i32, i32>(move _2) -> [return: bb1, unwind unreachable]; // scope 1 at $DIR/lower_intrinsics.rs:+1:14: +1:48
```

Comment:
Note that this also allows directly reading a `&`, though it's not used in `library` in *this* PR.